src/canvasAnimationHelpers.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var canvasAnimationHelpers = {
+ fade: function(image, targetOpacity, callback) {
```

Comment:
Seems like this should just be part of CanvasObject rather than being its own module.